src/test_utils.jl

```diff
@@ -88,7 +88,7 @@ distribution (with zero-vector mean and identity covariance matrix) for each obs
 function FakeTemplate{DistributionEstimate, MultiOutput}()
     FakeTemplate{DistributionEstimate, MultiOutput}() do num_variates, inputs
         std_dev = ones(num_variates)
-        return [MultivariateNormal(std_dev) for _ in 1:size(inputs, 2)]
+        return [Product(Normal.(0, std_dev)) for _ in 1:size(inputs, 2)]
```

Comment:
do we need to change the fakes?
I worry that some compositional models do require that the component is a gaussian.
And that then we won't be able to use the fakes to test them anymore.

Reply:
Which compositional models?

Reply:
> do we need to change the fakes?

Our Fakes would have allowed for this bug fix to regress. Making the fakes some random distribution is a good idea in that regard.

Reply:
Actually non will do that becuase a distribution is never an input, i think it is fine